<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>965</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>499</y>
      <width>951</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTextBrowser" name="textBrowser"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>0</y>
      <width>951</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTextEdit" name="text">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>4</number>
       </property>
       <widget class="QWidget" name="lex_tab">
        <attribute name="title">
         <string>Table of lexems</string>
        </attribute>
        <widget class="QTableView" name="lexTable">
         <property name="geometry">
          <rect>
           <x>-5</x>
           <y>1</y>
           <width>471</width>
           <height>461</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="state_tab">
        <attribute name="title">
         <string>Table of states</string>
        </attribute>
        <widget class="QTableView" name="statesTable">
         <property name="geometry">
          <rect>
           <x>-5</x>
           <y>1</y>
           <width>471</width>
           <height>461</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="conversion_tab">
        <attribute name="title">
         <string>Ð¡onversion Table</string>
        </attribute>
        <widget class="QTableView" name="conversionTable">
         <property name="geometry">
          <rect>
           <x>-5</x>
           <y>1</y>
           <width>471</width>
           <height>461</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="grammar_tab">
        <attribute name="title">
         <string>Grammar</string>
        </attribute>
        <widget class="QTableView" name="grammarTable">
         <property name="geometry">
          <rect>
           <x>5</x>
           <y>1</y>
           <width>461</width>
           <height>451</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="analysis_tab">
        <property name="focusPolicy">
         <enum>Qt::TabFocus</enum>
        </property>
        <attribute name="title">
         <string>Analysis and Poliz</string>
        </attribute>
        <widget class="QTableView" name="analysisTable">
         <property name="geometry">
          <rect>
           <x>-5</x>
           <y>1</y>
           <width>471</width>
           <height>461</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="poliz_tab">
        <attribute name="title">
         <string>Poliz Table</string>
        </attribute>
        <widget class="QTableView" name="polizTable">
         <property name="geometry">
          <rect>
           <x>-5</x>
           <y>1</y>
           <width>471</width>
           <height>461</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>965</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionCut"/>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <widget class="QMenu" name="menuProgram">
    <property name="title">
     <string>Program</string>
    </property>
    <addaction name="actionRun"/>
    <addaction name="actionRun_Relation_Table"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuProgram"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="recources.qrc">
     <normaloff>:/images/icons/New.bmp</normaloff>:/images/icons/New.bmp</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset>
     <normalon>:/images/icons/Open.bmp</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset>
     <normalon>:/images/icons/Save.bmp</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="text">
    <string>Run</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="recources.qrc">
     <normaloff>:/images/icons/Save.bmp</normaloff>:/images/icons/Save.bmp</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionRun_Relation_Table">
   <property name="text">
    <string>Run Relation Table</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="recources.qrc"/>
 </resources>
 <connections/>
</ui>
